.github/workflows/build.yml

```diff
@@ -3,26 +3,26 @@ on: [pull_request, workflow_dispatch]
 jobs:
   cocoapods:
     name: CocoaPods
-    runs-on: macOS-latest
+    runs-on: macOS-11.0
     steps:
       - name: Check out repository
         uses: actions/checkout@v2
-      - name: Use Xcode 12.4
-        run: sudo xcode-select -switch /Applications/Xcode_12.4.app
+      - name: Use Xcode 12.5
```

Comment:
For the sake of consistency, we should update `release.yml` to use Xcode 12.5 as well.